codepulse/src/main/scala/com/secdec/codepulse/data/model/slick/SlickH2ProjectDataProvider.scala

```diff
@@ -42,10 +42,10 @@ class SlickH2ProjectDataProvider(folder: File, actorSystem: ActorSystem) extends
 	private val cache = collection.mutable.Map.empty[ProjectId, SlickProjectData]
 
 	private object ProjectFilename {
-		def apply(projectId: ProjectId) = s"${getDbName(projectId)}.h2.db"
```

Comment:
Why was the filename changed? Is this related to Java 8 support?

My concern is: if we're changing the database filename, this will affect backward compatibility with existing data. We should probably try to not break that.

Reply:
I share your concern about backward compatibility. The newer version of the H2 driver is unrelated to Java 8 support. It is a dependency of dependency-check version 3.0.0.2, which was committed on 11/16, a day before Code Pulse version 1.1.15 got released.